docs/WebsocketAPIV4.md

```diff
@@ -0,0 +1,235 @@
+## WebSocket protocol v4 details
+
+WebSocket Protocol v4 requires an Exasol version of at least 8.31.0. It follows the RFC 6455 document of the IETF.
+
+The Exasol connection server identifies the initial GET request by the client.
+This request contains information about the used protocol version.
+Depending on this information the matching login and protocol class is
+chosen.
+
+After the handshake the process is identical to a connection using the
+standard drivers like JDBC or ODBC: The connection server listens to
+incoming messages and forwards the requests to the database. 
+
+## Table of contents
+* [Changes](#changes)
+* [Command summary](#command-summary)
+* [Attributes: Session and database properties](#attributes-session-and-database-properties)
+* [Data Types: Type names and properties](#data-types-type-names-and-properties)
+* [Compression](#compression)
+* [Heartbeat/Feedback messages](#heartbeatfeedback-messages)
+* [Subconnections](#subconnections)
+
+## Changes
+
+| Date | Exasol Version | Change | Issue |
+| --- | --- | --- | --- |
+| 2024.09.05 | 8.31.0 | The latest update to Exasol enables the use of the database port for subconnections instead of requiring ports in the range 20000 to 21000. Additionally, this update eliminates the need for special IP address configurations on the database. | |
```

Comment:
If DB Exa to Exa changes are merged this week, then DB version will be 8.32.0.

Reply:
Done